dashboards/lampy/lampy.json

```diff
@@ -485,7 +485,7 @@
             "id": "viz.markdown"
           },
           "rawConfiguration": {
-            "text": "![Python](https://s3.dualstack.us-east-2.amazonaws.com/pythondotorg-assets/media/community/logos/python-logo-only.png)"
+            "text": "![Python](https://raw.githubusercontent.com/newrelic/newrelic-quickstarts/ec054ad274f7ad248184911c445f2c711917076e/quickstarts/python/python/logo.svg)"
```

Comment:
@RamanaReddy8801 This logo is not showing full image so replace this?

Reply:
In the dashboard I am able to see full logo with this image, please have a look below:

![Screenshot 2023-05-31 at 10 20 28 PM](https://github.com/newrelic/newrelic-quickstarts/assets/106150577/f20db153-09cc-416a-b891-78911cda41dc)
![Screenshot 2023-05-31 at 10 20 09 PM](https://github.com/newrelic/newrelic-quickstarts/assets/106150577/36a2e0ee-2643-4ddf-b462-c31949365403)